WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -678,9 +679,10 @@ private void startObserving() {
         });
         mEditorMedia.getSnackBarMessage().observe(this, event -> {
             SnackbarMessageHolder messageHolder = event.getContentIfNotHandled();
-            if (messageHolder != null) {
+            if (messageHolder != null && messageHolder.getMessage() instanceof UiStringRes) {
```

Comment:
Is there any reason why we can accept only UiStringRes and not UiStringText? Have we considered using `mUiHelpers.getTextOfUiString(this, messageHolder.getMessage())`. ?

Reply:
Fixed in 2dcad11